applications/plugins/SofaPython3/ExternalProjectConfig.cmake.in

```diff
@@ -2,8 +2,8 @@ cmake_minimum_required(VERSION 3.11)
 
 include(ExternalProject)
 ExternalProject_Add(SofaPython3
-    GIT_REPOSITORY https://github.com/sofa-framework/SofaPython3
-    GIT_TAG origin/master
+    GIT_REPOSITORY https://github.com/fredroy/SofaPython3
+    GIT_TAG origin/follow_sofa_pr1834
```

Comment:
I see no PR on https://github.com/sofa-framework/SofaPython3 :eyes:

Reply:
I was planning to create PR once everything compiles (and everything is OK) but why not yes